code/game/machinery/computer/syndicate_specops_shuttle.dm

```diff
@@ -23,7 +23,7 @@ GLOBAL_VAR_INIT(syndicate_elite_shuttle_timeleft, 0)
 
 /proc/syndicate_elite_process()
 	var/area/syndicate_mothership/control/syndicate_ship = locate()//To find announcer. This area should exist for this proc to work.
-	var/area/syndicate_mothership/elite_squad/elite_squad = locate()//Where is the specops area located?
+	//var/area/syndicate_mothership/elite_squad/elite_squad = locate()//Where is the specops area located?
```

Comment:
Uh?

Reply:
This was commented out because the only line that used it was `elite_squad.readyreset()` after the commented out **Marauder** (syndies dont even get those, and its for the CC launchers) launch code was commented out. The area has 0 other reference.